actors/player.gd

```diff
@@ -1,14 +1,25 @@
-extends KinematicBody
+extends Actor
 
 export (int) var speed = 20
 export (int) var gravity = 20
 
 const DEAD_ZONE := 0.15
 const use_keyboard := true
+const actor_type = "player"
```

Comment:
Oisko parempi tän sijaan antaa tälle `class_name Player`, jollon voi kysyä `target.actor_type == 'player'` sijaan `target is Player`, jollon ei tarvis muistaa et näillä pitää olla tämmönen `actor_type` muuttuja. Ihan mielipidekysymys. Henkilökohtasesti pitäisin tota class name systeemiä vähän siistimpänä.